yfinance/base.py

```diff
@@ -375,7 +375,8 @@ def history(self, period="1mo", interval="1d",
         df = df[~df.index.duplicated(keep='first')]
         self._history = df.copy()
         if not actions:
-            df = df.drop(columns=["Dividends", "Stock Splits", "Capital Gains"])
+            columns_to_be_dropped = ["Dividends", "Stock Splits", "Capital Gains"]
+            df = df.drop([colname for colname in columns_to_be_dropped if colname in df.columns], axis=1)
```

Comment:
You can set `drop(errors='ignore')` instead. Neater code.

Reply:
Sure will do it